components/OrderBook/index.tsx

```diff
@@ -448,7 +462,7 @@ const OrderBookContainer = styled.div`
     display: flex;
     flex-direction: column;
     position: relative;
-    padding: 0.6rem 0;
+    padding: 0 0 0.6rem;
```

Comment:
I
![image](https://user-images.githubusercontent.com/29347276/126920245-5fb05ee2-c22d-4d3a-8d61-8a0f40409451.png)
causes the title to appear quite squished with no padding above